docs/abi-spec.rst

```diff
@@ -74,7 +74,7 @@ The following elementary types exist:
 
 The following (fixed-size) array type exists:
 
-- ``<type>[M]``: a fixed-length array of ``M`` elements, ``M >= 0``, of the given type.
+- ``<type>[M]``: a fixed-length array of ``M`` elements, ``M > 0``, of the given type.
```

Comment:
I think we had an argument and technically the ABI encoding allows for 0-element arrays to be described, just the compiler doesn't do that.

Reply:
OK. But that sounds like something that should be mentioned in the spec. I removed this change and added a note instead.